lab-lindsay/sll.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const Node = require('./model/list-node');
+
+//[val, next];
+//(HEAD) [val, next] This list is empty so point the head at the new Node
+
+//(HEAD) [val, next] => [val, next] => [val, next] => [val, next] => [val, next]
+
+//Prepend() adds a new node at the HEAD   append() adds a new node after the last Node
+
+const SLL = module.exports = function() {
+  this.head = null;
+  this._length = 0;
+};
+
+SLL.prototype.prepend = function(value) {
+  let node = new Node(value);
+  if(!this.head) {
+    this.head = node;
+    this._length++;
```

Comment:
keeping track of the length is a good idea. im curious as to why you decided to implement this? for me, it'd be nice to have that info so after a removal i could test the length property :)